server/src/test/java/org/opensearch/cluster/action/shard/ShardStateActionTests.java

```diff
@@ -112,16 +112,16 @@ private static class TestShardStateAction extends ShardStateAction {
             super(clusterService, transportService, allocationService, rerouteService, THREAD_POOL);
```

Comment:
https://github.com/opensearch-project/OpenSearch/blob/a87cede9b88976e0a043e7ea8670d3de2f400c1e/server/src/test/java/org/opensearch/cluster/action/shard/ShardStateActionTests.java#L499
setUpMasterRetryVerification -> setUpClusterManagerRetryVerification

Reply:
Changed the name in a new commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/37a0b17d4f96acd479401afa1bda51f15cf2d89f. Thank you!